core/func_ref.cpp

```diff
@@ -77,6 +88,8 @@ void FuncRef::_bind_methods() {
 		ClassDB::bind_vararg_method(METHOD_FLAGS_DEFAULT, "call_func", &FuncRef::call_func, mi, defargs);
 	}
 
+	ClassDB::bind_method(D_METHOD("call_funcv", "arguments"), &FuncRef::call_funcv);
```

Comment:
In Object the argument is called `arg_array`, it would be good to be consistent.